testsuite/features/reposync/srv_add_rocky8_repositories.feature

```diff
@@ -49,15 +49,6 @@ Feature: Add the Rocky 8 distribution custom repositories
     And I select the "rocky-8-iso-baseos" repo
     And I click on "Save Repositories"
     Then I should see a "repository information was successfully updated" text
-    And I wait until the channel "rocky-8-iso" has been synced
```

Comment:
if it syncs automatically, then wouldn't it make sense to wait until it's been synced anyway? :thinking:

Reply:
nevermind, it's the next scenario

Reply:
Yes the next scenario basically checks for that :wink: